docs/SUMMARY.md

```diff
@@ -0,0 +1,34 @@
+* [Home](index.md)
+* [Installation](installation.md)
+* Tutorial
+    * [Introduction to Ibis](tutorial/01-Introduction-to-Ibis.ipynb)
+    * [Aggregating and Joining](tutorial/02-Aggregates-Joins.ipynb)
+    * [Lazy Mode and Logging](tutorial/03-Expressions-Lazy-Mode-Logging.ipynb)
+    * [More Value Expressions](tutorial/04-More-Value-Expressions.ipynb)
+    * [Creating and Inserting External Data](tutorial/05-IO-Create-Insert-External-Data.ipynb)
+    * Advanced Topics
+        * [Complex Filtering](tutorial/06-ComplexFiltering.ipynb)
+        * [Analytics Tools](tutorial/07-Analytics-Tools.ipynb)
+        * [Geospatial Analysis](tutorial/08-Geospatial-Analysis.ipynb)
+* [Ibis for SQL Programmers](ibis-for-sql-programmers.ipynb)
+* [User Guide](user_guide/)
+* [Backends](backends/)
```

Comment:
Shall we rename it to `Execution Backends` or something less vague?

Reply:
Yep, makes sense.